core/templates/dev/head/base.html

```diff
@@ -53,9 +53,9 @@
     <meta property="og:url" content="{{FULL_URL}}">
     <meta property="og:description" content="{{meta_description}}">
     <meta property="og:type" content="article">
-    <meta property="og:image" content="{{DOMAIN_URL}}/images/logo/288x288_logo_mint.png">
+    <meta property="og:image" content="{{get_complete_static_resource_url(DOMAIN_URL, '/assets/images/logo/288x288_logo_mint.png')}}>
 
-    <link rel="apple-touch-icon" href="/images/logo/favicon.png">
+    <link rel="apple-touch-icon" href="{{get_static_resource_url('/assets/images/logo/favicon.png')}}">
```

Comment:
`GLOBALS` are not set until line 74 in this file, and without it we cannot interpolate using AngularJs. So, should I move the script tag above the meta tags or keep using jinja for meta tags?